src/main/modules/state-redirect/index.ts

```diff
@@ -28,6 +28,10 @@ export class StateRedirectMiddleware {
 
     app.use(
       errorHandler(async (req: AppRequest, res: Response, next: NextFunction) => {
+        if ([WEBCHAT_URL].includes(req.path as PageLink)) {
+          return next('route');
```

Comment:
See https://expressjs.com/en/guide/using-middleware.html and find "next('route')" on the page